### ISSUE TITLE
Motorcycle License

### ISSUE THREAD

**Issue by nelsonic** [nelsonic/nelsonic.github.io#328]
I've often said that if I ever got a motorcycle I wouldn't ride my (pedal) bike again...
Like most humans I have a tendency toward _laziness_ and if I had a motorbike I would _probably_ end up using it most of the time.
So, with that in mind, I'm **_not_** proposing **buying** a **motorcycle**!
_However_, I am thinking of getting a motor cycle license so that I can go for the odd "day trip" with my **future** [wife](https://github.com/dwyl/phase-two/issues/40) and **father-in-law** (_who is a major bike-lover and has ridden all over Europe!_)

https://www.confused.com/on-the-road/learning-to-drive/getting-your-motorbike-licence

@iteles given that you don't have a UK driving License or much _time_ for that matter, I propose that only I get the bike license _for now_. You will be able to ride _with_ me and if you ever get sick of being a "_passenger_" you can always get your own license... 😉 

Could get one of these for cash and go on an epic road trip: 
![image](https://cloud.githubusercontent.com/assets/194400/24580389/6096765c-16ff-11e7-90e0-11d0382e4bc1.png)
Joking aside a reliable motorbike would be a _really_ good form of transport in [`#PhaseTwo`](https://github.com/dwyl/phase-two) ...

**Comment by nelsonic** [comment]
Going to go to: http://www.eastsiderider.co.uk/
Looks like the CBT is £125 and the Theory Exam is £25
`.then` depending on the number of lessons required it could be a couple of hundred ...
`.then` the exam is £95 so all in should be about £500 ... 
which seems like a lot for something I'm _rarely_ going to use. 
but it might come in handy.

**Comment by nelsonic** [comment]
@iteles we don't _appear_ to have anything scheduled for the 28th April 2018:
![image](https://user-images.githubusercontent.com/194400/38859404-d81e51b6-4225-11e8-9c79-3551bd6e15a7.png)

Can I book my CBT for that day?

**Comment by iteles** [comment]
A Saturday? Sure. We don't have anything booked.

**Comment by nelsonic** [comment]
@iteles just _checking_ if we have any _guests_ or you have _plans_ for anything?

**Comment by nelsonic** [comment]
@iteles yeah, the training _lasts_ the "whole day" so a Saturday is the only day I can do ...
just _checking_ if we have any _guests_ or you have _plans_ for anything?

**Comment by iteles** [comment]
Nothing I can think of that would keep you from doing the day.
